pkg/daemon/gpsd.go

```diff
@@ -187,20 +220,25 @@ retry:
 		} else {
 			return // do not send event yet
 		}
-		processCfg.EventChannel <- event.EventChannel{
+		select {
+		case g.processConfig.EventChannel <- event.EventChannel{
 			ProcessName: event.GNSS,
 			State:       g.state,
-			CfgName:     processCfg.ConfigName,
+			CfgName:     g.processConfig.ConfigName,
 			IFace:       g.gmInterface,
 			Values: map[event.ValueType]int64{
 				event.GPS_STATUS: int64(tpv.Mode),
 				event.OFFSET:     g.offset,
 			},
-			ClockType:  processCfg.ClockType,
-			Time:       time.Now().Unix(),
+			ClockType:  g.processConfig.ClockType,
+			Time:       time.Now().UnixMilli(),
```

Comment:
Why is this UnixMilli and above is UnixNano?

Reply:
shoud be all UnixMili , will fix if any missing